storagenode/storagenodedb/storageusage.go

```diff
@@ -123,22 +119,49 @@ func (db *storageusageDB) GetDailyTotal(ctx context.Context, from, to time.Time)
 	var stamps []storageusage.Stamp
 	for rows.Next() {
 		var atRestTotal float64
-		var timeStamp time.Time
+		var intervalStart time.Time
 
-		err = rows.Scan(&atRestTotal, &timeStamp)
+		err = rows.Scan(&atRestTotal, &intervalStart)
 		if err != nil {
 			return nil, err
 		}
 
 		stamps = append(stamps, storageusage.Stamp{
-			AtRestTotal: atRestTotal,
-			Timestamp:   timeStamp,
+			AtRestTotal:   atRestTotal,
+			IntervalStart: intervalStart,
 		})
 	}
 
 	return stamps, nil
 }
 
+// Summary returns aggregated storage usage across all satellites
+func (db *storageusageDB) Summary(ctx context.Context, from, to time.Time) (_ float64, err error) {
+	defer mon.Task()(&ctx)(&err)
+	var summary sql.NullFloat64
```

Comment:
cannot it just be `float64`?

According the query it doesn't seem that it can be nil.

Reply:
It was float before, but I got an error at some point

Reply:
it sounds weird

Reply:
It could be a `var summary *float64` and then check for the `nil`.

But, yes, when there is no result it will fail to unmarshal into a `float64`.

Reply:
@egonelbre if we know that on empty result we won't to return `0` and no error, then `sql.NullTime` let us avoid extra check. Also it seems that it is preferred approach since we have this type provided by `sql` pkg

Reply:
@rikysya, sounds good.